schemas/common/b2b-contact.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-contact",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Contact",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Contact specific attributes.",
+  "definitions": {
+    "b2b-contact": {
+      "properties": {
+        "xdm:ID": {
```

Comment:
you can't use this - it's reserved for other things.   Use contactID

Reply:
@lrosenthol  - I may agree with you here, but what is it "reserved" for.

Reply:
I thought that you had reserved ID, because of the XED `_` conversions...

Reply:
@lrosenthol  - No, there is no problem with xdm:id in XED - it simply becomes a field named "id". The only id field which is changed is the "@id" which becomes "_id". Having said that I still have no problem with this becoming "contactId" so it has some meaning to the type of id it is.